README.adoc

```diff
@@ -781,6 +781,8 @@ true
 
 showing that the field **b** of each container object is now referencing the same `ByteBuffer` instance.
 
+An exception to this is when the field value implements `Resettable` interface, when the default value of the field is not null, and the current field value is of exactly the same class as the default value. In this case, `value.reset()` will be called instead of overwriting it with default value by reference.
+
 In order to work around this, if necessary, the marshallable class can override `Marshallable.reset`:
```

Comment:
Can we change an example below, do demonstrate, that now it's possible to change field class to `Resettable`, in order to:

1. Avoid having an undesired side-effect on reset.
2. Support a garbage-free behaviour out-of-the-box.